### PR TITLE
[@tests] reduce calls to wtr in IE workflow

### PR DESCRIPTION
**Pull request by taylor-vann**
Reduce calls to web test runner in IE github workflow

### CODE REVIEW

File: packages/tests/package.json [hunk 1]
```diff
@@ -45,6 +47,18 @@
       "dependencies": [
         "build"
       ]
```
Comment (aomarks) [inline packages/tests/package.json:49]:
```suggestion
      "dependencies": [
        "test:dev",
        "test:prod"
      ]
```

(And remove "command", I couldn't put that in the suggestion)

File: .changeset/olive-crews-complain.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'tests': patch
+---
+
+expose dev and prod test commands
```
Comment (aomarks) [inline .changeset/olive-crews-complain.md:5]: Remove changeset, not user facing

File: packages/tests/package.json (hunk 1 above)
Comment (aomarks) [reply packages/tests/package.json:49]: (this is optional, just a neater way to write the same thing)

**Review by aomarks — APPROVED** [review]

File: .changeset/olive-crews-complain.md (hunk 2 above)
Comment (taylor-vann) [reply .changeset/olive-crews-complain.md:5]: sounds good!

File: packages/tests/package.json (hunk 1 above)
Comment (taylor-vann) [reply packages/tests/package.json:49]: will do!

File: .changeset/olive-crews-complain.md (hunk 2 above)
Comment (taylor-vann) [reply .changeset/olive-crews-complain.md]: done :)

File: packages/tests/package.json (hunk 1 above)
Comment (taylor-vann) [reply packages/tests/package.json]: done!

Comment (aomarks) [reply packages/tests/package.json]: Make sure to also remove the command, or else it will run everything twice.

Comment (aomarks) [reply packages/tests/package.json]: I still see the command

File: .github/workflows/sauce-ie11.yaml [hunk 3]
```diff
@@ -23,6 +23,12 @@ jobs:
           node-version: 16
           cache: 'npm'
           cache-dependency-path: '**/package-lock.json'
+      
+      - uses: saucelabs/sauce-connect-action@v2
```
Comment (aomarks) [inline .github/workflows/sauce-ie11.yaml:27]: This shouldn't be here, right?

**Review by aomarks — CHANGES_REQUESTED** [review]
Seeing a new change that doesn't seem to belong

File: .github/workflows/sauce-ie11.yaml (hunk 3 above)
Comment (taylor-vann) [reply .github/workflows/sauce-ie11.yaml:27]: yes but that's from a totally different commit

Comment (taylor-vann) [reply .github/workflows/sauce-ie11.yaml:27]: good catch but i have no idea why that's there

Comment (taylor-vann) [reply .github/workflows/sauce-ie11.yaml]: done

**Comment by taylor-vann** [comment]
kewl, removed artifacts.
also removed the added wireit commands since this doesn't need them anymore

**Review by aomarks — APPROVED** [review]

**Pull request merged**